config/locales/de/views/account_links.yml

```diff
@@ -1,6 +1,9 @@
 ---
 de:
   account_links:
+    common:
+      proforma_version:
+        select_default: aktuellste Version (momentan %{latest_version})
```

Comment:
Please ensure to align the English and German locale. The German would translate to "latest version", whereas the one you committed is only "latest" (without "version").